client/src/components/utils/option.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Select } from "antd";
+
+const { Option } = Select;
+
+const Options = typeValues => {
+  const selectOption = typeValues.map(({ key, value }) => (
+    <Option children={value} key={key} value={value} />
```

Comment:
@AbdallahAmmar96 React will automatically convert child content in the jsx into a parent's `children` prop, which means you could also write it like this:
```
<Option key={key} value={value} />
  {children}
</Option>
```